setup.py

```diff
@@ -109,6 +109,7 @@ def get_git_sha():
         "sqlalchemy-utils>=0.33.2,<0.36.5",
         "sqlparse>=0.3.0, <0.4",
         "wtforms-json",
+        "xlrd>=1.2.0",
```

Comment:
Unfortunately, it's not possible, because of `pandas` use `xlrd` and without it, it will return this error:
```
Missing optional dependency 'xlrd'. Install xlrd >= 1.0.0 for Excel support Use pip or conda to install xlrd.
```

Reply:
Let's make this dependency optional by placing it in `setup.py` under `extras_require`, something like
```
extras_require={
    "excel": ["xlrd>=1.2.0,xlrd<1.3"],
}
```